src/guide/reviewing-and-sponsoring-a-change.md

```diff
@@ -8,16 +8,24 @@
+If you are new to an area, reviewing changes is also a great way to learn the code and see what general styles and types of changes that are relevant in the area. Be mindful though, if you don't know the area well you should state this in your review comments.
 
 New developers in the OpenJDK community don't have the permissions needed to integrate changes to the repositories. This is a feature that ensures that all developers get familiar with the code, processes, and community before being allowed to actually make changes. To get their first changes in, the new Contributor needs the help of a Sponsor. The Sponsor's role is to offer constructive advice and eventually integrate the sponsored contribution into the repository.
 
 Sponsoring new contributions is an important activity - it's how the engineering culture of a project gets passed on from the core group to new Contributors, from one generation to the next. Take pride in the value you provide to Contributors. Their success reflects well on you.
 
-As a **R**eviewer, Contributors will look up to you for guidance to get their contributions into the project - your actions will determine whether Contributors will feel welcome and want to engage further with the project beyond their initial attempt, or not. Let's not lose enthusiastic, engaged and technically competent Contributors due to a lack of communication. If you see a request in your area of expertise and you can't address it, at least acknowledge receipt of the request and provide an estimate for when you'll be able to give it your attention. A frank explanation of your time constraints or commitments will be appreciated and respected.
-
 There are many different reasons to sponsor a change and depending on the situation the exact steps involved may differ significantly. An experienced developer hired into a larger team of OpenJDK developers is likely to get much of the required information by osmosis and will likely need less help from a sponsor than the enthusiast sitting at home with no prior experience of large scale software development. This text aims to highlight the steps involved in sponsoring a change but can't cover all possible scenarios. As always, common sense takes precedence where the assumptions made here doesn't apply to your use case.
 
+## Responsibilities of a Reviewer
+
+As a [Reviewer](https://openjdk.org/bylaws#reviewer) you have a responsibility to make sure changes are sound and aline with the general direction of the area. If you, as a [Reviever](https://openjdk.org/bylaws#reviewer), review a change in an area that you don't know well you probably shouldn't be the one to approve the change.
```

Comment:
s/aline/align/

Reply:
Just to expand here though, a change can often touch on multiple areas and require reviews by multiple people. It is common to give a partial review (for your area) but also to click the "approve" button to indicate approval for that area. Authors will typically (informally) collect sufficient reviews to cover all aspects of the change - or more formally you can bump the number of needed reviewers to match the number of different areas.

Reply:
Typo fixed.